.github/workflows/publishing.yml

```diff
@@ -905,11 +905,11 @@ jobs:
       # https://packaging.python.org/en/latest/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/
       - name: Create release
         run: |
-          for installer in `ls installers/`; do 
-            mv "installers/$installer" "$installer"
+          for installer in `find installers/ -type f`; do 
+            mv -v "$installer" "$(basename -- "$installer")"
```

Comment:
Question just because I'm curious: is the `--` truly needed? I was thinking it would only be needed if the `$installer` started with a hyphen for some reason, but maybe I'm missing something.

Reply:
I don't think it is needed, especially with the quotes. In general, it is useful for the case you mention but also to avoid command injection vulnerabilities.